probe_ble_specification.rst

```diff
@@ -268,7 +268,7 @@ Value                     Format   Bytes Description
 ========================= ======== ===== ======================================
 Sequence number           uint32_t 4     Sequence number of the record.
 Raw temperature data      uint8_t  13    See `raw temperature data`_.
-Virtual sensors and state uint16_t 2     See `Virtual Sensors and State Log`_.
+Virtual sensors and state uint8_t  7     See `Virtual Sensors and State Log`_.
```

Comment:
Maybe consider the name "Prediction Log" instead of "Virtual Sensors and State Log".